cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/readJsonFile/instancesConfigurationsJsonFile/PricingReader.java

```diff
@@ -0,0 +1,46 @@
+package org.cloudsimplus.sla.readJsonFile.instancesConfigurationsJsonFile;
```

Comment:
Renamed the class to AwsEC2InstanceReader.

Add a method as below to create a VM from the configuration read:

public Vm createVm()

This way, you can use the method to create EC2-like VMs in your experiments.